mail_debrand/models/mail_render_mixin.py

```diff
@@ -22,6 +22,19 @@ def remove_href_odoo(
         if type(value) is bytes:
             value = value.decode()
         has_odoo_link = re.search(r"<a\s(.*)odoo\.com", value, flags=re.IGNORECASE)
+        extra_regex_to_skip = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("mail_debrand.extra_regex_to_skip", False)
+        )
+        # value is required field on ir config_parameter, so we have added
+        # safety check for "False"
+        if has_odoo_link and extra_regex_to_skip != "False" and extra_regex_to_skip:
+            # check each regex to be skipped
+            for regex in extra_regex_to_skip.split(","):
```

Comment:
Why do you split the regex expression by a comma? You can also just use a `|` right (standard regex operator)?